### PR TITLE
Config with pure python

### PR DESCRIPTION
**Pull request by icemac**
* Lint the code 
* Add support for Python 3.7, 3.8 and 3.9.
* Drop support for Python 3.3 and 3.4.
* Bring coverage to 100 % for branch coverage.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -67,45 +69,46 @@ def read(*rnames):
...
+          'Programming Language :: Python :: 3.8',
+          'Programming Language :: Python :: 3.9',
+          'Programming Language :: Python :: Implementation :: CPython',
+          'Programming Language :: Python :: Implementation :: PyPy',
+          'Natural Language :: English',
+          'Operating System :: OS Independent',
+          'Topic :: Internet :: WWW/HTTP',
+          'Framework :: Zope :: 3',
+      ],
+      url='http://github.com/zopefoundation/zope.app.applicationcontrol',
```
Comment (mgedmin) [inline setup.py]:
```suggestion
      url='https://github.com/zopefoundation/zope.app.applicationcontrol',
```

File: README.rst [hunk 2]
```diff
@@ -6,20 +6,3 @@ The application control instance is usually generated upon startup.
...
-    http://svn.zope.org/zope.app.applicationcontrol/
-
-- Read-only Subversion checkout::
-
-  $ svn co svn://svn.zope.org/repos/main/zope.app.applicationcontrol/trunk
-
-- Writable Subversion checkout::
-
-  $ svn co svn+ssh://userid@svn.zope.org/repos/main/zope.app.applicationcontrol/trunk
-
```
Comment (mgedmin) [inline README.rst:25]: Haha we still had this :D

**Review by mgedmin — APPROVED** [review]
LGTM

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**